staging/src/k8s.io/kubectl/pkg/generate/versioned/service.go

```diff
@@ -210,6 +211,15 @@ func generateService(genericParams map[string]interface{}) (runtime.Object, erro
 			service.Spec.Ports[i].TargetPort = intstr.FromInt(int(port))
 		}
 	}
+	if params["node-port"] != "" {
```

Comment:
I am wondering if shouldn't be:
```
if len(params["node-port"]) != 0 {
```

Reply:
Thanks! Yes, existing lines are written with such a manner. Fixed.